docs/content/configuration.md

```diff
@@ -25,30 +25,63 @@ context first, then explain how to use multiple contexts thereafter.
 Settings can be configured via Grizzly itself, using `grr config set`. They are stored
 in an OS specific location.
 
-## Grafana Itself
+## Authenticate with a Grafana instance
 
 Grizzly interacts with Grafana via its REST API. For this, you will need to
 establish authentication credentials.
 
+The minimum requirement is to set the URL of the Grafana instance to be used:
 ```sh
 grr config set grafana.url http://localhost:3000 # URL for the root of your Grafana instance
-grr config set grafana.user admin # (Optional) Username if using basic auth
+```
+
+Optionally, set the following field(s), depending our your [authentication method with the given Grafana instance](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/):
+- A [token](#token-or-password-optional) if using a [Grafana service account](https://grafana.com/docs/grafana/latest/administration/service-accounts) (recommended)
+- A [username](#username-optional) and [password](#token-or-password-optional) if using basic authentication
+
+Next, consider setting a [context](#using-grizzly-contexts) to save this configuration.
+
+Once you have configured your authentication method, you are ready to use the [Grizzly server](./server.md) to view and/or edit resources.
+
+### Token or password (optional)
+
+To authenticate with a Grafana instance using a service account token, first [create a service account in the
+Grafana instance](https://grafana.com/docs/grafana/latest/administration/service-accounts/#to-create-a-service-account).
+Before clicking `Create` to create the service account, ensure that you have added the role that
+best suits your use case. A role with `Viewer` permissions would be enough
+to view resources. Use `Editor` or `Admin` to be able to edit resources.
+Then,
+[add a token to the new service account](https://grafana.com/docs/grafana/latest/administration/service-accounts/#add-a-token-to-a-service-account-in-grafana).
+Copy the token and set it with Grizzly:
+
+```sh
 grr config set grafana.token abcd12345 # Service account token (or basic auth password)
 ```
 
-## Grafana Cloud Prometheus
-To interact with Grafana Cloud Prometheus (aka Mimir), use these settings:
+Alternatively, set this field to a password if using [basic authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/#basic-authentication).
+
+### Username (optional)
+
+Optionally, set a username if using [basic authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/#basic-authentication).
+
+```sh
+grr config set grafana.user admin # (Optional) Username if using basic auth
+```
+
+## Authenticate with hosted Prometheus
+
+To interact with [hosted Prometheus / Mimir](./prometheus.md) resources, use these settings:
```

Comment:
I replaced `Grafana Cloud Prometheus` with `hosted Prometheus` so that it's clearer that this is specifically for Prometheus/Mimir. That way folks don't wonder whether they need to do this if they want to use Grafana Cloud ☺️